opta/commands/secret.py

```diff
@@ -178,7 +179,45 @@ def update(
     update_manual_secrets(layer.name, {secret: str(value)})
     __restart_deployments(no_restart, layer.name)
 
-    print("Success")
+    logger.info("Success")
+
+
+@secret.command()
+@click.argument("secret")
+@restart_option
+@config_option
+@env_option
+@input_variable_option
+@local_option
+def remove(
+    secret: str,
+    env: Optional[str],
+    config: str,
+    no_restart: bool,
+    local: Optional[bool],
+    var: Dict[str, str],
+) -> None:
+    """Update a given secret of a k8s service by removing a manual entry
+
+    Examples:
+
+    opta secret remove -c my-service.yaml "MY_SECRET_1"
+    """
+
+    config = check_opta_file_exists(config)
+    if local:
+        config = local_setup(config, input_variables=var)
+        env = "localopta"
+    layer = Layer.load_from_yaml(config, env, input_variables=var)
+    gen_all(layer)
+
+    set_kube_config(layer)
+    create_namespace_if_not_exists(layer.name)
```

Comment:
If the namespace doesn't exist, do we really need to create it if the goal is to delete a secret?